lib/timescaledb/model/peer_metrics.ex

```diff
@@ -7,40 +7,44 @@ defmodule Membrane.RTC.Engine.TimescaleDB.Model.PeerMetrics do
 
   import Ecto.Changeset
 
+  alias Membrane.RTC.Engine.TimescaleDB.Model.TrackMetrics
+
   @type t :: %__MODULE__{
-          time: NaiveDateTime.t() | nil,
-          peer_id: String.t() | nil,
-          "peer.metadata": String.t() | nil,
+          id: integer(),
+          peer_id: String.t(),
           "ice.binding_requests_received": non_neg_integer() | nil,
           "ice.binding_responses_sent": non_neg_integer() | nil,
           "ice.bytes_received": non_neg_integer() | nil,
           "ice.bytes_sent": non_neg_integer() | nil,
           "ice.packets_received": non_neg_integer() | nil,
-          "ice.packets_sent": non_neg_integer() | nil
+          "ice.packets_sent": non_neg_integer() | nil,
+          tracks_metrics: [TrackMetrics] | nil,
+          inserted_at: DateTime.t()
         }
 
-  @primary_key false
+  @primary_key {:id, :id, autogenerate: true}
```

Comment:
This is actually the default value, can be omitted
https://hexdocs.pm/ecto/Ecto.Schema.html#module-schema-attributes